docs/sources/mimir/set-up/migrate/migrate-from-thanos-or-prometheus.md

```diff
@@ -69,6 +69,63 @@ Blocks from Prometheus don't have any external labels stored in them.
 Only blocks from Thanos use labels.
 {{< /admonition >}}
 
+{{< admonition type="note" >}}
+If you encounter the HTTP error 413 "Request Entity Too Large" when uploading blocks using mimirtool, and if Nginx is being used as a reverse proxy, the uploaded block size may be exceeding Nginx's default maximum allowed request body size.
+To resolve this issue:
+
+Determine the current size of the blocks you are trying to upload by running the following command:
+
+```bash
+find <path/to/blocks> -name 'chunks' -printf '%s\n' | numfmt --to=iec-i
+```
+
+This will show the size of each block's chunks directory in a human-readable format.
+Increase the client_max_body_size directive in the Nginx configuration:
+
+For manual Nginx deployments, open the Nginx configuration file (e.g., /etc/nginx/nginx.conf) and set the client_max_body_size directive inside the server block for the Mimir endpoint to a value about 5% larger than the maximum size of the blocks you are uploading. For example:
+
+```
+server {
+    ...
+    client_max_body_size 95M;
+    ...
+}
+```
+
+```
+location / {
+    ...
+    client_max_body_size 95M;
+    ...
+}
+```
+
+For Helm deployments of Mimir, you can set the `gateway.nginx.config.clientMaxBodySize` value in your Helm values file to a higher value, e.g.:
+
+```yaml
+gateway:
+  nginx:
+    config:
+      clientMaxBodySize: 95M
+```
```

Comment:
[chunk files are capped at 512MiB by default](https://github.com/grafana/backend-enterprise/blob/7674f7923b76e524c5b282a1747e933b2a91aaf4/vendor/github.com/prometheus/prometheus/tsdb/chunks/chunks.go#L301-L302). What do you think about just having `540M` as the recommended value and omit the `find` command?

Reply:
I believe the same thing is also set for prometheus
540M seems like a good idea